raiden/network/proxies/secret_registry.py

```diff
@@ -88,61 +120,175 @@ def register_secret_batch(
             return
 
         checking_block = self.client.get_checking_block()
-        error_prefix = 'Call to registerSecretBatch will fail'
         gas_limit = self.proxy.estimate_gas(checking_block, 'registerSecretBatch', secrets)
+        receipt = None
+        transaction_hash = None
+        msg = None
+
         if gas_limit:
-            error_prefix = 'Call to registerSecretBatch failed'
+            gas_limit = safe_gas_limit(
+                gas_limit,
+                len(secrets) * GAS_REQUIRED_PER_SECRET_IN_BATCH,
+            )
+
+            log.debug('registerSecretBatch called', **log_details)
+
             try:
-                gas_limit = safe_gas_limit(
+                transaction_hash = self.proxy.transact(
+                    'registerSecretBatch',
                     gas_limit,
-                    len(secrets) * GAS_REQUIRED_PER_SECRET_IN_BATCH,
```

Comment:
This was a bug, we don't want to send `secrets` but `secrets_to_register`

Reply:
arh nice catch for all the `secrets` -> `secrets_to_register` palces